dataplane/dregress.cpp

```diff
@@ -4,19 +4,45 @@
 #include "common/fallback.h"
 
 #include "checksum.h"
-#include "controlplane.h"
 #include "dataplane.h"
 #include "dregress.h"
 #include "metadata.h"
-#include "worker.h"
+#include "slow_worker.h"
+
+dregress_t::dregress_t(dataplane::SlowWorker* slow, cDataPlane* dataplane, uint32_t gc_step) :
+        slow_worker_{slow},
+        dataplane{dataplane},
+        stats{},
+        connections{new dregress::ConnTable},
+        gc_step{gc_step}
+{}
+dregress_t::dregress_t(dregress_t&& other) :
+        connections{nullptr}
+{
+	*this = std::move(other);
+
+	other.connections = nullptr;
+}
 
-dregress_t::dregress_t(cControlPlane* controlplane,
-                       cDataPlane* dataplane) :
-        controlplane(controlplane),
-        dataplane(dataplane)
+dregress_t& dregress_t::operator=(dregress_t&& other)
 {
-	memset(&stats, 0, sizeof(stats));
-	connections = new dataplane::hashtable_chain_spinlock_t<dregress::connection_key_t, dregress::connection_value_t, YANET_CONFIG_DREGRESS_HT_SIZE, YANET_CONFIG_DREGRESS_HT_EXTENDED_SIZE, 4, 4>();
+	slow_worker_ = other.slow_worker_;
+	dataplane = other.dataplane;
+	stats = other.stats;
+	std::swap(connections, other.connections);
```

Comment:
when `connections` will be `std::unique_ptr` instead of raw pointer, use `std::move` here